plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -63,13 +70,23 @@ const addToCartWithOptionsStore = store(
 					{ lock: universalLock }
 				);
 
+				const { productId } = getContext< Context >();
```

Comment:
Below in lines 79-83 you access the context again, especially getting the `productId` is duplicated. What about moving those lines here so it's consolidated?

Reply:
Ouch. :man_facepalming:  Good catch with that! Improved it in 0a987468fc04a8d48d92882ee3cc0ca4544e384c.